helpers.lua

```diff
@@ -570,9 +573,10 @@ function serve(uri, cache)
     elseif ext == "ms" then
         local data = get_data_for(file)
         content = lustache:render(content, data)
-    elseif ext == "json" then
+    elseif uri == "/ynh_userinfo.json" then
```

Comment:
Why don't we do the same thing for other json files?

Reply:
Cause if i want to put a .json in a theme containing other kind of data i don't want to get the context data but my original json file.